python/sdss_install/utils/module.py

```diff
@@ -64,7 +64,11 @@ def set_modules_home(self):
                 self.modules_home = {'dir': modules_home}
                 self.modules_home['lmod'] = join(modules_home, "libexec", "lmod")
                 self.modules_home['lua'] = join(modules_home, "init", "lmodrc.lua")
-                self.modules_home['tcl'] = join(modules_home, "libexec", "modulecmd.tcl")
+                tcl = []
+                tcl.append(join(modules_home, "modulecmd.tcl"))
+                tcl.append(join(modules_home, "libexec", "modulecmd.tcl"))
+                for self.modules_home['tcl'] in tcl:
+                    if exists(tcl): break
```

Comment:
If neither `join(modules_home, "modulecmd.tcl")` nor `join(modules_home, "libexec", "modulecmd.tcl")` exist then `self.modules_home['tcl']` will be set to the latter. Is this the desired behavior? If so would the following be more readable?
'''
tcl = join(modules_home, "modulecmd.tcl")
self.modules_home['tcl'] = tcl if exists(tcl) else join(modules_home, "libexec", "modulecmd.tcl")
'''